### ISSUE TITLE
Lots of accessories missing

### ISSUE THREAD

**Issue by Programie** [Sroose/homebridge-loxone-ws#10]
I've just tested Homebridge with your Loxone plugin and it is pretty nice.

But in the Home app, I'm missing lots of accessories. In my case, there is only one Blinds/Jalousie item and there are also not all light items available.

The console output of Homebridge also does not list the missing items, so I think that's an issue with the plugin.

**Comment by DerFlash** [comment]
I think I've found the issue and I've the same problem here.
In the ItemFactory.js the parsed items are saved within the itemList using their name. So as soon as you name your Lamps just "Light" in Loxone (which should be fine since you've already sorted them by their room attribute), you'll only get one of them recognised by the Homebridge.

I've tried to use their uuid as a key or just add them by Array.push and changing the for-loop, but then I'll get an error and unfortunately I've no time to debug this yet:
> Jun 29 13:36:45 raspberrypi homebridge[2703]: throw new Error("Cannot add a bridged Accessory with the same UUID as another bridged Accessory: " + existing.UUID);
